netman/adapters/switches/brocade.py

```diff
@@ -515,8 +527,8 @@ def __init__(self, *args, **kwargs):
         super(VlanBrocade, self).__init__(*args, **kwargs)
 
         self.vlan_interface_name = kwargs.pop('vlan_interface_name', None)
-        self.tagged_interfaces = []
         self.icmp_redirects = True
+        self.has_no_untagged_interface = False
```

Comment:
if the statement is here,m the solution is to show vlan id... and read the actual interfaces, so basically that information is discarded and of no use